From: Quinlan Pfiffer <qpfiffer@gmail.com>
Date: Fri, 28 Sep 2012 19:58:44 +0000
Subject: asix: Adds support for Lenovo 10/100 USB dongle.

commit 66dc81ecd71332783c92fb170950d5ddb43da461 upstream.

This dongle ships with the X1 Carbon, and has an AX88772B
usb to ethernet chip in it.

Signed-off-by: Quinlan Pfiffer <qpfiffer@gmail.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
[bwh: Backported to 3.2: adjust filename]
---
 drivers/net/usb/asix.c |    4 ++++
 1 file changed, 4 insertions(+)

Index: linux-3.2.46/drivers/net/usb/asix.c
===================================================================
--- linux-3.2.46.orig/drivers/net/usb/asix.c	2013-07-26 18:53:46.000000000 +0000
+++ linux-3.2.46/drivers/net/usb/asix.c	2013-07-26 19:25:43.000000000 +0000
@@ -1616,6 +1616,10 @@
 	USB_DEVICE (0x04f1, 0x3008),
 	.driver_info = (unsigned long) &ax8817x_info,
 }, {
+	// Lenovo U2L100P 10/100
+	USB_DEVICE (0x17ef, 0x7203),
+	.driver_info = (unsigned long) &ax88772_info,
+}, {
 	// ASIX AX88772B 10/100
 	USB_DEVICE (0x0b95, 0x772b),
 	.driver_info = (unsigned long) &ax88772_info,
